index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7bfe064e41.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio Page</title>
+</head>
+<body>
+ <header>
+ <h1 class="my_logo"><a href="#">Atril</a></h1>
+ <nav>
+    <div class="bar"></div>
+    <div class="bar"></div>
+    <div class="bar"></div>
+ </nav>
+ </header>
+<section class="container">
+<section class="header-text">
+    <h2>I’m Atril <br>
+        Glad to see you!</h2>
```

Comment:
- [Optional] Kindly, this part is your main heading so use an `h1` tag for search engines. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 👍👌